packages/message-kit/src/lib/core.ts

```
@@ -96,11 +135,14 @@
   awaitingResponse: boolean = false;
-  awaitedHandler: ((text: string) => Promise<boolean | void>) | null = null;
-  getMessageById!: (id: string) => DecodedMessage | null;
-  executeSkill!: (text: string) => Promise<SkillResponse | undefined>;
-  clearMemory!: (address?: string) => Promise<void>;
-  clearCache!: (address?: string) => Promise<void>;
+  getMessageById: (id: string) => DecodedMessage | null = () => null;
+  executeSkill: (text: string) => Promise<SkillResponse | undefined> =
+    async () => undefined;
+  clearMemory: (address?: string) => Promise<void> = async () => {};
+  clearCache: (address?: string) => Promise<void> = async () => {};
+  awaitedHandler: ((text: string) => Promise<boolean | void>) | undefined =
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)